### ISSUE TITLE
List of available nodes

### ISSUE THREAD

**Issue by mscansian** [n8n-io/n8n#33]
When I first started using this library the main information I wanted was to know what services were already implemented. I could not find this information anywhere. I know someone can just browse through the source code and see what's available, but I think this is not easy for someone who is seeing the project for the first time.

My suggestion is to include another section in the README listing all available integrations (and maybe some actions or examples). It's the responsibility of the developer that creates a node to update this table.

My goal with this is to give a glance to someone new that n8n has a lot of integrations in place and is the right tool to what they are looking for.

PS: Maybe it would be better to add this information in the documentation (maybe even create a single page for every node or service), but in this case a link in the README directly to this page is really important.

**Comment by janober** [comment]
Sorry are not in front of my computer so can just answer very short from my phone.

More or less what you describe above can be found here:
https://n8n.io/nodes

It reads the data from the `n8n-nodes-base` packages.

However is also still a wip and hope to have soon more time to improve that page.

**Comment by mscansian** [comment]
@janober It would be nice if we put a link to this in the README, don't you think?

**Comment by janober** [comment]
Actually one is already there but apparently not placed very good.

**Comment by joshavant** [comment]
Hi. I was just now trying to evaluate if n8n was going to be useful for my needs.

I spent a handful of minutes searching n8n documentation for a simple listing of available nodes. After failing to find this info, I went to Google, searched, and found this thread.

The link above (https://docs.n8n.io/integrations/) is exactly what I was looking for.

I'd suggest that maybe this issue still needs some work. :)

Perhaps https://docs.n8n.io/integrations/ should be linked to from one or both of these pages:
- https://docs.n8n.io/integrations/
- https://docs.n8n.io/integrations/builtin/
